### ISSUE TITLE
use mix of ICD10 and ICD9 for createPhenotypes?

### ISSUE THREAD

**Issue by brettva** [PheWAS/PheWAS#54]
We have a system that switched from ICD9 to ICD10 at a transition date, as a result some of my cohort has ICD9 (earlier samples) and some have ICD10 codes (later samples). 

Can I provide my whole cohort containing this mixture of ICD9 and ICD10 to the createPhenotypes function at one time so long as I appropriately code "ICD9CM" or "ICD10CM" in the `vocabulary_id` column of the `id.vocab.code.index` argument? Or does createPhenotypes work using only exclusively ICD9 or exclusively ICD10 at one time?

Thanks!

**Comment by RobertJCarroll** [comment]
Yes, the default accepts both ICD9CM and ICD10CM codes simultaneously.
